ovos_core.md

```diff
@@ -16,6 +16,10 @@
 | mycroft.ready                    | None                               | Signals that Mycroft is ready               | None                                  |
 | mycroft.skills.trained           | None                               | Indicates that skills have been trained     | None                                  |
 | ovos.setup.finished              | None                               | Indicates the completion of the setup process | None                                |
+| skillmanager.activate            | {"skill_id": string}               | Activates a skill, if it is available       | skillmanager.activate.response        |
```

Comment:
can we replace description with loads/unloads instead of activate/deactivates?

want to start reducing confusion around what "active" is, let's keep that wording reserved to the Session object concept